lib/stat_tracker.rb

```diff
@@ -5,35 +5,17 @@
 class StatTracker
   attr_reader :games, :teams, :game_teams
 
-  def initialize(games, teams, game_teams)
-    @games = games
-    @teams = teams
-    @game_teams = game_teams
+  def initialize(games_file, teams_, game_teams)
```

Comment:
Changed the instance variables here to match what the collection classes are expecting, mostly for readability